scripts/format_source.py

```diff
@@ -0,0 +1,11 @@
+import json
+from typing import List, cast
+
+
+def format_source(source: List[str]) -> List[str]:
+    """If source from basemaps cli it has a string which contains a list that needs to be split
```

Comment:
this is actually a Argo constraint that baseamaps, Argo only passes JSON objects into sub commands 😭